logstash-core/src/test/java/org/logstash/config/ir/imperative/IfStatementTest.java

```diff
@@ -0,0 +1,25 @@
+package org.logstash.config.ir.imperative;
+
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+
+import org.logstash.config.ir.InvalidIRException;
+import org.logstash.config.ir.graph.Graph;
+
+import static org.logstash.config.ir.IRHelpers.*;
+
+public class IfStatementTest {
+
+    @Test
+    public void testEmptyIfStatement() throws InvalidIRException {
```

Comment:
I realize that this file didn't exist previously. We primarily tested this path through other means.

That said, this feels a bit complete with only the test for the new case we've added.

Would you mind adding one more test for the case where the if statement does have non-noop paths?

Reply:
No problem at all, happy to add more tests here!